src/redux/categories/categories.js

```diff
@@ -0,0 +1,17 @@
+// Action types
+const CHECK_STATUS = 'CHECK_STATUS';
```

Comment:
- [ ] (OPTIONAL)  Kindly make sure that the values of your action type follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux).